src/plugins/robot-memory/event_trigger_manager.cpp

```diff
@@ -82,8 +82,7 @@ void EventTriggerManager::check_events()
   for(EventTrigger *trigger : triggers)
   {
     try {
-      while(trigger->oplog_cursor->more())
-      {
+      while(trigger->oplog_cursor->more()) {
```

Comment:
It's still missing the space in front of the parenthesis. Sorry, but  since it was mentioned before, let me be picky here. Please fix once more.

Reply:
Yes, thanks, I missed that.